src/mlpack/tests/hmm_test.cpp

```diff
@@ -1664,26 +1667,26 @@ BOOST_AUTO_TEST_CASE(DiagonalGMMHMMMultipleGaussiansUnlabeledTrainingTest)
   for (size_t i = 0; i < 2; ++i)
   {
     // Check the estimated weights using the second DiagonalGMM.
-    BOOST_REQUIRE_SMALL(hmm.Emission()[1].Weights()[sortedIndices[i]] -
-        gmms[1].Weights()[i], 0.08);
+    REQUIRE(hmm.Emission()[1].Weights()[sortedIndices[i]] -
+        gmms[1].Weights()[i] == Approx(0.0).margin(0.08));
 
     // Check the estimated means using the second DiagonalGMM.
-    BOOST_REQUIRE_LT(arma::norm(
+    REQUIRE(arma::norm(
       hmm.Emission()[1].Component(sortedIndices[i]).Mean() -
-      gmms[1].Component(i).Mean()), 0.35);
+      gmms[1].Component(i).Mean()) < 0.35);
 
     // Check the estimated covariances using the second DiagonalGMM.
-    BOOST_REQUIRE_LT(arma::norm(
+    REQUIRE(arma::norm(
       hmm.Emission()[1].Component(sortedIndices[i]).Covariance() -
-      gmms[1].Component(i).Covariance()), 0.6);
+      gmms[1].Component(i).Covariance()) < 0.6);
   }
 }
 
 /**
  * Make sure the labeled training works reasonably given multiple distributions
  * with diagonal covariance.
  */
-BOOST_AUTO_TEST_CASE(DiagonalGMMHMMMultipleGaussiansLabeledTrainingTest)
+TEST_CASE("DiagonalGMMHMMMultipleGaussiansLabeledTrainingTest", "[HMMTest]")
 {
   math::RandomSeed(std::time(NULL));
```

Comment:
Oh, oops, this line should not be here!  I must have left it here while debugging.  Can you remove that while you're modifying this file?  Thanks! :+1: